README.md

```diff
@@ -21,7 +20,7 @@ queue.startPolling(
     callback();
   },
   function error(queueError) {
-    throw queueError;
+    console.error(queueError);
```

Comment:
... or because this is intended to be `required` within `sawmill`, I suggest this follow the rule of silence and simply pass the error on.